runtime-infra/pn-oer-dashboard.yaml

```diff
@@ -49,84 +49,65 @@ Resources:
-
+  
+          const CSS = `<style>td,th{font-family:Arial,sans-serif;font-size:14px;white-space:nowrap;text-align:left;padding:3px;border-bottom:1px solid #f2f2f2}td:first-child{text-align:left}tr:hover{background:#fbf8e9!important;transition:all .1s ease-in-out}tbody tr:nth-child(even){background:#f5f5f5;box-shadow:0 1px 0 rgba(255,255,255,.8) inset}th{text-align:left;text-shadow:0 1px 0 rgba(255,255,255,.5);border-bottom:1px solid #ccc;background-color:#dce9f9;background-image:linear-gradient(top,#ebf3fc,#dce9f9)}th:first-child{border-radius:6px 0 0 0}th:last-child{border-radius:0 6px 0 0}th:only-child{border-radius:6px 6px 0 0}</style>`;
+  
           exports.handler = async (event) => {
-            
+  
             if (event.describe) {
-              return DOCS;   
-            }
-
-            const tables = []
-
-            const dynamoTables = await getDynamoTablesSize()
-            for(let i=0; i<dynamoTables.length; i++){
-              tables.push(dynamoTables[i]);
+              return DOCS;
             }
+  
+            const tables = [];
+            const dynamoTables = await getDynamoTablesSize();
             
             const initialValue = 0;
-            const totalDynamoSize = tables.reduce(
+            const totalDynamoSize = dynamoTables.reduce(
               (accumulator, currentValue) => accumulator + currentValue.size,
               initialValue
             );
-            
-            tables.push({
+  
+            dynamoTables.push({
               name: 'Total DynamoDB Size',
               size: totalDynamoSize
-            })
-
-            const metricRows = tables.map((resource, i) => {
+            });
+              
+            const metricRows = dynamoTables.map((resource, i) => {
               let html = `<tr><td><b>${resource.name}</b></td>`;
               html += `<td>${Number(resource.size/(1024*1024*1024)).toFixed(6)} GB</td></tr>`;
               return html;
             });
-              
-            const tableStart = `<table class="cwdb-no-default-styles"><thead><tr><th>Table Name</th><th>Used Storage</th></tr></thead>`
-            return CSS + tableStart + `<tbody>${metricRows.join('')}</tbody></table>`;    
+            
+            const tableStart = `<table class="cwdb-no-default-styles"><thead><tr><th>Table Name</th><th>Used Storage</th></tr></thead>`;
+            return CSS + tableStart + `<tbody>${metricRows.join('')}</tbody></table>`;
           };
+
       Description: "Storage function custom widget"
       FunctionName: pn-storage-function
       Handler: index.handler
       MemorySize: 128
       Role: !GetAtt StorageFunctionIAMRole.Arn
-      Runtime: nodejs16.x
+      Runtime: nodejs20.x
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You must specify a valid value for Runtime (nodejs20.x). Valid values are [&#34;dotnet6&#34;, &#34;dotnetcore1.0&#34;, &#34;dotnetcore2.0&#34;, &#34;dotnetcore2.1&#34;, &#34;dotnetcore3.1&#34;, &#34;go1.x&#34;, &#34;java11&#34;, &#34;java17&#34;, &#34;java8&#34;, &#34;java8.al2&#34;, &#34;nodejs&#34;, &#34;nodejs10.x&#34;, &#34;nodejs12.x&#34;, &#34;nodejs14.x&#34;, &#34;nodejs16.x&#34;, &#34;nodejs18.x&#34;, &#34;nodejs4.3&#34;, &#34;nodejs4.3-edge&#34;, &#34;nodejs6.10&#34;, &#34;nodejs8.10&#34;, &#34;provided&#34;, &#34;provided.al2&#34;, &#34;python2.7&#34;, &#34;python3.10&#34;, &#34;python3.11&#34;, &#34;python3.6&#34;, &#34;python3.7&#34;, &#34;python3.8&#34;, &#34;python3.9&#34;, &#34;ruby2.5&#34;, &#34;ruby2.7&#34;, &#34;ruby3.2&#34;]